chips/earlgrey/src/spi_host.rs

```diff
@@ -0,0 +1,10 @@
+use kernel::utilities::StaticRef;
+use lowrisc::spi_host::SpiHostRegisters;
+
+//These addresses have been changed in mainline of OT
+//Refer: https://github.com/lowRISC/opentitan/blob/c4f342b9349ba033a5f22fba9349999299a1b2bf/hw/top_earlgrey/sw/autogen/top_earlgrey_memory.h#L179
```

Comment:
Do you mean they have been changed since the commit we use for OT?

Reply:
Yep since the commit, I will remove this comment cause it's all up-to date now. I put it there as a *note to self*